src/ucp/core/ucp_rkey.c

```diff
@@ -18,14 +19,58 @@ static ucp_rkey_t ucp_mem_dummy_rkey = {
 
 static ucp_md_map_t ucp_mem_dummy_buffer = 0;
 
+
+ucs_status_t ucp_rkey_write(ucp_context_h context,
```

Comment:
maybe call it ucp_rkey_pack_uct() or something similar?

Reply:
renamed